src/runner/main.cpp

```diff
@@ -243,7 +169,8 @@ enum class SpecialMode
 {
     None,
     Win32ToastNotificationCOMServer,
-    ToastNotificationHandler
+    ToastNotificationHandler,
+    ReportSuccessfulUpdate

```

Comment:
What do we do if the update fails for any reason? Do we notify the user?

Reply:
currently we don't. just retrying to update after 24hrs

Reply:
We can merge this PR but then open an issue to deal with this, since it could be a recurring failure, after one or max two failures we should notify the user and offer to visit the download page to install it manually.

Reply:
sure, https://github.com/microsoft/PowerToys/issues/2275#issue-603772705